services/motion/builtin/move_on_globe_test.go

```diff
@@ -286,7 +286,7 @@ func TestObstacleReplanningGlobe(t *testing.T) {
 	}
 
 	extra := map[string]interface{}{"max_replans": 10, "max_ik_solutions": 1, "smooth_iter": 1, "motion_profile": "position_only"}
-	extraNoReplan := map[string]interface{}{"max_replans": 0, "max_ik_solutions": 1, "smooth_iter": 1}
+	extraNoReplan := map[string]interface{}{"max_replans": 0, "max_ik_solutions": 1, "smooth_iter": 1, "motion_profile": "position_only"}
```

Comment:
I had to change this to include the position only motion profile since otherwise we would have the following at the end of the run:

```
spatialmath.OrientationAlmostEqual(goal.Pose().Orientation(), basePose.Orientation()):  false
spatialmath.PoseAlmostCoincidentEps(goal.Pose(), basePose, motionCfg.planDeviationMM):  true
```


I am not entirely sure why this is the case since at plan time we should have considered this.
Please lmk if you have any thoughts on if this is a bug, or if the `atGoalCheck` success criteria conditional is set-up incorrectly.

Reply:
```
goal.Pose().Orientation().OrientationVectorDegrees():  &{0 0 0 1}
basePose.Orientation().OrientationVectorDegrees():  &{-2.0007803398817288 0 0 1}
```

It seems we were off by 2 degrees which seems negligible enough for me..
Lmk what you think.
It makes sense to me that there is a 2 degree difference since we are no longer running tests on fake components.

Do we think the 2 degree difference is indicative of something else that is wrong in our stack?

Reply:
A 2-degree difference here seems completely reasonable to me

Reply:
I will change it to be 
```
spatialmath.OrientationAlmostEqualEps(goal.Pose().Orientation(), basePose.Orientation(), 5)
```
so we have 5 degrees of breathing room